### DISCUSSION TITLE
Add "Google Discover" in Sources

### DISCUSSION THREAD

**Discussion by Dockers** [plausible/analytics#1889] [Anything Else]
Hi,
Do you think it's possible to add "Google Discover" in Sources list ? I guess trafic from Google Discover go in "Direct" today.
The referrer is `android-app://com.google.android.googlequicksearchbox`
Thank for your time.

**Comment by metmarkosaric** [comment]
thanks for the feedback @Dockers! that would be nice for sure! there's already a thread on this topic: https://github.com/plausible/analytics/discussions/1337
